extras/dcgoss/README.md

```diff
@@ -9,7 +9,7 @@ containers. It is based on `dgoss`.
 
```

Comment:
lets revert the changes in this file, since it's dcgoss aka docker-compose goss.

Reply:
Done.